packages/flutter_app/lib/features/cookbook/chat_ui/chat_message_bubble.dart

```diff
@@ -1,3 +1,4 @@
+// ignore_for_file: avoid_hardcoded_color
```

Comment:
I've added an ignore comment.
```
sentAtStyle: effectiveTextStyle.copyWith(
        color: Theme.of(context).colorScheme.brightness == Brightness.dark
            ? Colors.black54
            : Colors.white54,
      ),
```
A warning message was displayed for the above part, but the Flutter app template doesn't have a chat screen page, and this is the only part where the color switching is done. Therefore, it doesn't seem necessary to set the color in the colorScheme.

Reply:
![badge](https://img.shields.io/badge/review-imo-important) 

This class(`ChatMessageBubble`) is not referenced anywhere, so I think it can be deleted

![badge](https://img.shields.io/badge/review-nits-inactive) 
I recommend ignoring warnings for specific lines rather than the entire file with ‘ignore for file’ because it may cause you to miss other important warnings.

Reply:
![badge](https://img.shields.io/badge/review-imo-important)

I see ChatMessageBubble is used in ChatAppConversationPage. The only problem is that the page is not accessible from the app.
It would be good to keep it as a Cookbook.

As for specifying colors, why not use ColorScheme?
Specify the color to change according to the theme.

```dart
      sentAtStyle: effectiveTextStyle.copyWith(
        color: Theme.of(context).colorScheme.onPrimary.withOpacity(0.5),
      ),
```

Reply:
Thanks for the suggestion.
I have made the change to display using colorScheme in this commit.

https://github.com/altive/flutter_app_template/pull/468/commits/553e6408c3b314d57d91a7dd1202e063c0be70bb